doc/ref_arch/openstack/chapters/chapter01.md

```diff
@@ -19,7 +19,7 @@ This Reference Architectures is focussed on OpenStack as the VIM chosen based on
 
 <a name="1.1.1"></a>
 ## 1.1.1. Vision
-The ultimate goal is that the OpenStack-based CNTT Reference Architecture will host NFV workloads, primarily VNFs, of interest to the Common NFVI Telco community.  Initially the OpenStack-based Reference Architecture would be used for testing and validating VNFs that can then be deployed in compatible operator clouds.   The Reference Architecture document can be used by operators to deploy compliant production infrastructure.  
+The OpenStack-based CNTT Reference Architecture will host NFV workloads, primarily VNFs, of interest to the CNTT community.  The Reference Architecture document can be used by operators to deploy CNTT conformant infrastructure.  
```

Comment:
Should we be avoiding the use of the temporary term "CNTT"?

Reply:
@iangardner22 CNTT is used throughout the documents -- RM, RA-1, ... This should be a TSC decision and if required to change should be made across all documents. Do you agree?

Reply:
Yes agree entirely!